site.region

```diff
@@ -423,14 +423,14 @@
                    title="Red Badger Google Plus"
                    class="site-footer__socialIcon"
                    >
-                  <img src="/assets/g-plus.svg" title="Google Plus" alt="Google plus logo">
+                  <img src="/assets/g-plus.svg" title="Google Plus" alt="Google Plus">
                 </a>
               </li>
             </ul>
             </div>
             <div class="site-footer__section site-footer__address site-footer__underline boldSansSerif fontXS">
               <div role="presentation" class="site-footer__mapPin">
```

Comment:
double check whether role: presentation apply to the child image tag

Reply:
It does 🙌 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role